task_14/js/task4.js

```diff
@@ -0,0 +1,23 @@
+let a = prompt('введите первое число');
+let e = prompt('введите операцию');
```

Comment:
I think name for this variable should be `operation`

Reply:
done